SignInCheckIn/SignInCheckIn/Services/RoomService.cs

```diff
@@ -35,5 +45,68 @@ public EventRoomDto CreateOrUpdateEventRoom(string authenticationToken, EventRoo
 
             return Mapper.Map<EventRoomDto>(response);
         }
+
+        public List<AgeGradeDto> GetEventRoomAgesAndGrades(string authenticationToken, int eventId, int roomId)
```

Comment:
I wrote the backend stuff, I'll bet you wouldn't have commented if you knew that... :) Haha, I'll split it up in the morning.